.github/workflows/payoutManagementSystem.yml

```diff
@@ -0,0 +1,44 @@
+name: Payout Management System
+
+on: [push, pull_request]
+
+jobs:
+  format:
+    name: Check format
+    runs-on: ubuntu-22.04
```

Comment:
what is the recommended version of ubuntu suggested by github?
what strategy of version selection is recommeded ?